website/tools.shtml

```diff
@@ -0,0 +1,55 @@
+<!--#set var="dj_page_title" value="Tools and extensions" --><!--#include virtual="/header.shtml" -->
+
+<p>
+    This page lists tools and facilities that can work with DOMjudge. They are
+    not in all cases maintained by the DOMjudge team.
+</p>
+
+
+<h2>Creating problemsets</h2>
+
+<h3>Checktestdata</h3>
```

Comment:
Do you want to also link to:
https://github.com/RagnarGrootKoerkamp/BAPCtools
https://github.com/Kattis/problemtools

Reply:
We accept packages created by either toolset, right? That seems like a good reason to list them.

Reply:
Yes, I'll provide a suggestion for both.